pkg/virt-operator/resource/apply/core_test.go

```diff
@@ -20,712 +20,710 @@
 package apply
 
 import (
-	"crypto/tls"
```

Comment:
I find it hard to review this commit and ensure that there is no unintended glitch. It would be easier for me if you split it to logic commit, whitespace commit and quoting commit. I don't really mind if someone else approves this as it is.

Reply:
@nunnatsa  please consider splitting this to facilitate quicker review

Reply:
+1
I'd be happy to revisit this when it's broken up to commits
/cc @iholder-redhat

Reply:
IDE formatting is different than gofmt. that caused the changes. fixed. Now only the right changes applied.